testgrid/config/config.yaml

```diff
@@ -820,6 +820,8 @@ test_groups:
   gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gke-large-teardown
 - name: kubernetes-update-jenkins-jobs
   gcs_prefix: kubernetes-jenkins/logs/kubernetes-update-jenkins-jobs
+- name: ci-kubernetes-e2e-gce-taint-evict
+  gcs_prefix: kubernetes-jenkins/logs/kubernetes-e2e-gce-taint-evict
```

Comment:
whoops.. I mean the log path(gcs_prefix) need to match job name.. testgrid tab name is arbitrary..

gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-taint-evict